dev/TeachingTip/TeachingTip.cpp

```diff
@@ -1307,8 +1370,8 @@ winrt::TeachingTipPlacementMode TeachingTip::DetermineEffectivePlacement()
             }
 
             // If the horizontal midpoint is out of the window.
-            if (targetBounds.X + (targetBounds.Width / 2) < MinimumTipEdgeToTailCenter() ||
-                targetBounds.X + (targetBounds.Width / 2) > windowBounds.Width - MinimumTipEdgeToTailCenter())
```

Comment:
Are we just removing the buffer here?

Reply:
This check makes sure that the point that the tail is pointing to is on screen. This check was a little over zealous before this change, it made sure that the tip's edge would still be within the window.  This aspect is already checked by the "If the tip is too wide to fit between the center of the target and the left edge of the window." and isn't appropriate for this check when you consider that the tip can go out of the window.

---
In reply to: [268776728](https://github.com/Microsoft/microsoft-ui-xaml/pull/480#discussion_r268776728) [](ancestors = 268776728)